celesta-maven-plugin/src/main/java/ru/curs/celesta/plugin/maven/CursorGenerator.java

```diff
@@ -410,6 +398,25 @@ private static MethodSpec buildParseResult(
         return builder.build();
     }
 
+    private static MethodSpec buildGetFieldValue() {
+        String nameParam = "name";
+
+        return MethodSpec.methodBuilder("_getFieldValue")
```

Comment:
I agree that current solution might be not the fastest one but it a) works b) I'm not aware of how it can be quickly written in Java Poet. Let's keep it as an idea for the future.